scripts/machine_translation/README.md

```diff
@@ -156,7 +158,7 @@ Test BLEU score with 3 seeds (evaluated via sacre BLEU):
 (test bleu / valid bleu)
 | Subword Model | #Params    | Seed = 123  | Seed = 1234 | Seed = 12345 |  Mean±std   |
 |---------------|------------|-------------|-------------|--------------|-------------|
-| yttm          |            | 26.50/26.29 | -           |  -           |  -          |
+| yttm          |            | 26.78/25.96 | -           |  -           |  -          |
```

Comment:
Also, let's remove the missing seeds + subword algorithms and only attach the results of yttm.